lib/open_project/costs/engine.rb

```diff
@@ -65,35 +65,35 @@ class Engine < ::Rails::Engine
       end
 
       # Menu extensions
-      menu :top_menu,
+      menu :admin_menu,
            :cost_types,
-           {:controller => '/cost_types', :action => 'index'},
-           :caption => :cost_types_title,
-           :if => Proc.new { User.current.admin? }
+           { controller: '/cost_types', action: 'index' },
+           html: { class: 'icon2 icon-tracker' },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.